releasenotes/notes/fix-profiler-gevent-tasks-76ed862362210cb7.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    profiling: correct accounting of wall and CPU time for gevent tasks within
+    the main Python thread.
```

nit: casing and tensing 
```suggestion
    profiling: Corrects accounting of wall and CPU time for gevent tasks within
    the main Python thread.
```